resqpy/fault/_grid_connection_set.py

```diff
@@ -1258,11 +1271,11 @@ def write_nexus_header_lines(fp, axis, polarity, fault_name, grid_name = 'ROOT')
                 tm.log_nexus_tm('warning')
             active_nexus_head = (axis, polarity, fault_name, grid_name)
 
-        def write_row(gcs, fp, name, i, j, k1, k2, axis, polarity):
+        def write_row(gcs, fp, name, i, j, k1, k2, axis, polarity, tmult):
             nonlocal row_count
             write_nexus_header_lines(fp, axis, polarity, name)
-            fp.write('\t{0:1d}\t{1:1d}\t{2:1d}\t{3:1d}\t{4:1d}\t{5:1d}\t1.0\n'.format(
-                i + 1, i + 1, j + 1, j + 1, k1 + 1, k2 + 1))
+            fp.write('\t{0:1d}\t{1:1d}\t{2:1d}\t{3:1d}\t{4:1d}\t{5:1d}\t{6:.2f}\n'.format(
```

Comment:
I think we need at least 3 decimal places for multipliers: Believe it or not, REs can want to set a multiplier of 0.001 and regard it as different to zero.  I suggest :.4f for the tmult format (instead of :.2f), to be on the safe side.